client/logmon/logmon.go

```diff
@@ -249,6 +261,7 @@ func (l *logRotatorWrapper) Close() {
 	closeDone := make(chan struct{})
 	go func() {
 		defer close(closeDone)
+		<-l.opened
```

Comment:
Let's always comment receives outside of a select. I think I know why this is safe (if for no other reason than because logmon is an external process so leaking this blocked goroutine isn't dangerous as it will get cleaned up when the process exits)